hummingbot/client/command/start_command.py

```diff
@@ -259,7 +258,7 @@ async def confirm_oracle_conversion_rate(self,  # type: HummingbotApplication
             self.placeholder_mode = True
             self.app.hide_input = True
             for pair in settings.rate_oracle_pairs:
-                msg = await RateCommand.oracle_rate_msg(pair)
+                msg = await self.oracle_rate_msg(pair)
```

Comment:
This change is because the HB APP already inherits the Rate command and you are cleaning the code right?